packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -112,6 +130,25 @@ public function renderStyleProp(Environment $environment, array $props): string
         ]);
     }
 
+    /**
+     * Converts a mixed value to a string.
+     *
+     * @return string The value as a string.
+     */
+    private function getSpacingClassName(string $propName, string $propValue, ?string $breakpoint = null): string
+    {
+        $container = new ContainerBuilder();
+        $classPrefix = $container->hasParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX)
+            ? $container->getParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX)
+            : '';
+        $classPrefix = is_string($classPrefix) ? $classPrefix : '';
+        $utilityName = self::SPACING_STYLE_PROPS[$propName];
+        $utilityValue = $propValue == self::STYLE_SPACING_AUTO ? self::STYLE_SPACING_AUTO : preg_replace('/[^0-9]/', '', $propValue);
```

Comment:
It will filter the number from the string, e. g: `abc123` -> `123`. The same way can be used `filter_var` function (https://www.php.net/manual/en/function.filter-var.php).